<launch>
  <arg name="stimuli_feedback_period" />
  <arg name="lavis_lights_controller" />
  <arg name="lavis_audio_controller" />
  <node name="lavis_stimuli_controller_node" pkg="lavis_stimuli_controller" type="lavis_stimuli_controller_node.py">
    <param name="feedback_period" value="$(arg stimuli_feedback_period)"/>

    <!-- remap output topics -->
    <remap from="cmd_current" to="/$(arg lavis_lights_controller)/cmd_current"/>
    <remap from="cmd_off" to="/$(arg lavis_lights_controller)/cmd_off"/>
    <remap from="cmd_all_off" to="/$(arg lavis_lights_controller)/cmd_all_off"/>
    <remap from="play_tone" to="/$(arg lavis_audio_controller)/play_tone"/>

  </node>
</launch>
